packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -391,9 +393,7 @@ class TimelineFrame {
   int get startTime =>
       cpuStartTime != null ? min(cpuStartTime, _startTime) : _startTime;
```

Comment:
this also looks like a nullSafeMin call. Same for 
get endTime.

Reply:
done.